demos/blast/blast_in_example.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <seqan/blast.h>
+
+using namespace seqan;
+
+int main()
```

Comment:
If you add a tutorial in the manual, please reuse this demo - don't make a new one.